app/javascript/channels/project_channel.ts

```diff
@@ -1,23 +1,28 @@
-import $ from "jquery"
 import consumer from "./consumer"
 
-consumer.subscriptions.create(
-  {
-    channel: "ProjectChannel",
-    organization_id: $("#organization_subscription_identifier").val(),
-  },
-  {
-    connected() {
-      // Called when the subscription is ready for use on the server
-    },
-    disconnected() {
-      // Called when the subscription has been terminated by the server
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
I think we need to wait for the DOM to be ready before subscribing; otherwise, we won't get the organization id rendered in the DOM.